### PR TITLE
Fix cross platform filepaths

### LINKED ISSUE #91: NPM package: When exporting as .gltf output path is incorrect
When exporting as .gltf using the `fbx2gltf` npm package the output path is incorrect. Files are placed in the `<outputPath>/<fileName>_out` directory but the output path is returned as `<outputPath>/<fileName>.gltf`.

### PR DESCRIPTION
**Pull request by robertlong**
I noticed that exporting a .gltf was breaking in a few different ways on Windows. There are a few places where Unix path separators were used so I fixed it so that it uses the platform's file separator.

Also the npm module wasn't returning the path to the .gltf file so I changed it so that it does.

I'm still testing this, I've noticed some issues where strings do not seem to be terminating correctly in STDOUT output.

Fixes #91

### CODE REVIEW

File: src/Raw2Gltf.cpp [hunk 1]
```diff
@@ -53,7 +53,7 @@ const static std::string defaultSceneName = "Root Scene";
  * reference counting type, but generally speaking we pass around simple T& and T* types because the GLTFData
  * struct will, by design, outlive all other activity that takes place during in a single conversion run.
  */
-template<typename T>
```
Comment (zellski) [inline src/Raw2Gltf.cpp]: I prefer all of these without a space.

File: src/Raw2Gltf.cpp [hunk 2]
```diff
@@ -76,7 +76,8 @@ struct GLTFData
     std::shared_ptr<BufferViewData> GetAlignedBufferView(BufferData &buffer, const BufferViewData::GL_ArrayType target)
     {
         unsigned long bufferSize = this->binary->size();
-        if ((bufferSize % 4) > 0) {
```
Comment (zellski) [inline src/Raw2Gltf.cpp]: I prefer the opening brace on the same line as the conditional. Actually, a lot of this diff is just code style changes. I'd prefer if we eliminated those (in most cases) from the PR.

File: src/Raw2Gltf.cpp [hunk 3]
```diff
@@ -570,43 +629,53 @@ ModelData *Raw2Gltf(
...
-
-            } else if (!relativeFilename.empty()) {
+            }
+            else if (!relativeFilename.empty())
+            {
                 image = new ImageData(relativeFilename, relativeFilename);
-                std::string outputPath = outputFolder + relativeFilename;
-                if (FileUtils::CopyFile(rawTexture.fileLocation, outputPath)) {
-                    if (verboseOutput) {
+
```
Comment (zellski) [inline src/Raw2Gltf.cpp]: Alright, I scanned all the changes in this file, and I **think** this tweak is the only thing that's not just whitespace. I'll manually merge this in a bit, working on that assumption.

**Comment by cainmartin** [comment]
Any action on this PR, would be really useful to get this merged. :)

**Comment by zellski** [comment]
@cainmartin I will try to reconstruct it for 0.9.6 release soon.

**Comment by zellski** [comment]
Continued in https://github.com/facebookincubator/FBX2glTF/pull/135.

**Pull request closed without merging**